node_modules/oae-core/signin/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+    "i18n": {
+        "default": "bundles/default.properties"
```

Comment:
This properties file appears to be empty, so it should be removed and the `i18n` section should be removed from this manifest